src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -19,16 +19,6 @@ spec:
 
   network:
     allow:
-      - description: "UDS Operator"
-        direction: Ingress
-        selector:
-          app.kubernetes.io/name: keycloak
-        remoteNamespace: pepr-system
-        remoteSelector:
-          app: pepr-uds-core-watcher
-        remoteServiceAccount: pepr-uds-core
-        port: 8080
-
```

Comment:
I think this is fine, since its covered by backchannel access (<insert always has been meme>). But just to call it out - Pepr still needs this ingress regardless of client registration method, it just is/has been covered by a different netpol.